docs/docs/shards/index.md

```diff
@@ -249,4 +249,59 @@ For example, `None` as one of the valid data types for `:Max` parameter in shard
     [info] [2022-07-22 13:45:03.293] [T-19992] [logging.cpp::55] [mywire] 311828859
     ```
 
+### Enum
+
+`Enum` stands for enumerated data type. 
+
+What this means is that the value you pass to an enumerated variable can only take certain 'states' or named constant values.
+
+For example, in [`(Math.Mean)`](https://docs.fragcolor.xyz/shards/Math/Mean/) the value for `:Kind` parameter needs to be of type `Enum`.
+
+```clojure linenums="1"
+(Math.Mean
```

Comment:
This is invalid code, need to specify right enum value.

Reply:
This is the function definition for `Math.Mean` (as displaye on its [docs page](https://docs.fragcolor.xyz/shards/Math/Mean/)) - the sample code is after this, from line no. 275.

If this is confusing, I'll remove the function definition blocks from all types in this PR (I've used them in every shard type section).

Reply:
Did you check the sample in the docs?
https://docs.fragcolor.xyz/shards/Math/Mean/

Reply:
`(Enum)` is just a logger formatter text. Which ends in the docs because we didn't expand. But it's not code. Check code under to see the actual enum used. We need to indeed expand the enums and custom objects in the docs. It has been ages but it was never done. Cc @Kryptos-FR

Reply:
> Did you check the sample in the docs? https://docs.fragcolor.xyz/shards/Math/Mean/

Yeah. I based my 1st example code (for arithmetic mean)

```
    [2.0 10.0]
    (Math.Mean :Kind Mean.Arithmetic)
    (Log)   ;; AM => 6
```
on the sample from our docs page https://docs.fragcolor.xyz/shards/Math/Mean/
```
(Const [-1.0 0.0 1.0 2.0 5.0])
(Math.Mean
 ;:Kind
 Mean.Arithmetic)
(Assert.Is 1.4 true)
```

The rest two of my example codes are for geometric and harmonic means.

Reply:
> `(Enum)` is just a logger formatter text. Which ends in the docs because we didn't expand. But it's not code. Check code under to see the actual enum used. We need to indeed expand the enums and custom objects in the docs. It has been ages but it was never done. Cc @Kryptos-FR

You mean like this, for example? - actual enums  for `(FS.Copy)`
https://github.com/fragcolor-xyz/shards/blob/devel/src/core/shards/fs.cpp#L329

I see 55 such shards.
Have created [issue #]() to document their enums.

Reply:
Yep 👍